AENS.md

```diff
@@ -186,13 +218,15 @@ to commit to a name and after the commitment has been accepted into the
 chain, reveal the name to finish the process.
 
 A commitment should be binding, i.e. the claimant cannot change
-the value they commited to withouth changing the actual commitment
+the value they commited to without changing the actual commitment
 and hiding, so that a malicious miner learns nothing about the value
 the claimant has commited until they chose to reveal that value. This
 prevents malicious miners from front running, i.e. upon seeing a
 claim transaction, including their own claim request for the same
 name instead of the original claimant's one.
 
+After the claim transaction auctioning the name is conducted in clear
```

Comment:
It doesn't make sense to me, maybe a comma missing somewhere?

Reply:
rephrased